Doc/using/cmdline.rst

```diff
@@ -939,8 +939,8 @@ conflict.
 .. envvar:: PYTHONEXECUTABLE
 
    If this environment variable is set, ``sys.argv[0]`` will be set to its
```

Comment:
I think this documentation is wrong as well, at least when I test this on Linux. I see the following behavior:

```bash
$ PYTHONEXECUTABLE=/dev/null ./python -c 'import sys;print(sys.argv[0], sys.executable, sep="\n")'
-c
/dev/null
```

Reply:
Good catch. Yeah, skimming the code, I don't see where it affects sys.argv at all; it only seems to factor into how sys.executable is decided. It affecting sys.argv[0] makes zero sense -- sys.argv carries the "user program" args, not the C-level argv[0] of the program path.

Changed to say it affects sys.executable, not sys.argv[0]